### PR TITLE
[10945] Adapt to latest VSS tools changes. Rename data_type to dataty…

### PR DESCRIPTION
**Pull request by dennismeister93**
…pe. Rename break_on_noncore_attribute to break_on_unknown_attribute

Signed-off-by: Dennis Meister <dennis.meister@bosch-softtec.com>

## Describe your changes

Adapting vehicle-model-generator to changes in https://github.com/COVESA/vss-tools/commit/088791c15ac6afc18389e6446c06d1e33bd62032#diff-bd7a7e19458ea183d09243c6b66711908091c977580d1564a1c9f4e6e7c0c63aL93

## Issue ticket number and link

[AB#_[10945]_](https://softwaredefinedvehicle.visualstudio.com/SoftwareDefinedVehicle/_workitems/edit/10945)
https://github.com/eclipse-velocitas/vehicle-model-generator/issues/4

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Review by emsameen — CHANGES_REQUESTED** [review]
I approve the changes, LGTM. However, I would move the status to pending until the contributor approval is ready and the source of the PR is adjusted.

File: gen_vehicle_model.py
Comment (MP91) [inline gen_vehicle_model.py:91]: the attribute is just a bool so we should set it to true
```suggestion
            break_on_unknown_attribute=True,
```

**Review by MP91 — CHANGES_REQUESTED** [review]

**Comment by dennismeister93** [comment]
Closed due to official PR from SDV org https://github.com/eclipse-velocitas/vehicle-model-generator/pull/6

**Pull request closed without merging**
